packages/govuk-frontend/src/govuk/components/select/template.test.js

```diff
@@ -61,6 +91,13 @@ describe('Select', () => {
       expect($selectedItem.attr('selected')).toBeTruthy()
     })
 
+    it('selects options with implicit value using selected value', () => {
+      const $ = render('select', examples['without values with selected value'])
+
+      const $selectedItem = $('option:contains(\'Green\')')
```

Comment:
ESLint should be happy with double quotes when they're intentionally nesting single quotes if you like

```suggestion
      const $selectedItem = $("option:contains('Green')")
```